jre/guide/java/java_jre_configure_crypto_policy/oval/shared.xml

```diff
@@ -0,0 +1,30 @@
+<def-group>
+  <definition class="compliance" id="java_jre_configure_crypto_policy" version="1">
+    <metadata>
+      <title>Configure JRE to use System Crypto Policy.</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
```

Comment:
I think there is no need to keep platform Fedora.
Platform `Java Runtime Environment` is applicable as long as java is installed.